tests/charts/test_scheduler.py

```diff
@@ -518,3 +518,11 @@ def test_log_groomer_resources(self):
                 "memory": "2Gi",
             },
         } == jmespath.search("spec.template.spec.containers[1].resources", docs[0])
+
+    def test_dags_persistence_subpath(self):
+        docs = render_chart(
+            values={"dags": {"persistence": {"enabled": True, "subPath": "test/dags"}}},
+            show_only=["templates/scheduler/scheduler-deployment.yaml"],
+        )
+
+        assert "dags" in jmespath.search("spec.template.spec.containers[0].volumeMounts[*].name", docs[0])
```

Comment:
This isn't a "good" assertion, as it would pass without any of your subpath template changes, or even without subPath in the values in this test. Try something like this instead:

```suggestion
        assert "test/dags" in jmespath.search("spec.template.spec.containers[0].volumeMounts[?name == 'dags'].subPath", docs[0])
```

Reply:
In fact, adding it to this test would be even better:
https://github.com/apache/airflow/blob/c063fc688cf20c37ed830de5e3dac4a664fd8241/tests/charts/test_airflow_common.py#L47